### DISCUSSION TITLE
Not able to connect to the Bing server?

### DISCUSSION THREAD

**Discussion by zakmar06** [kubilus1/autoortho#187] [Q&A]
![image](https://github.com/kubilus1/autoortho/assets/106925138/3724b8bb-4d99-48d6-92cd-26e9e0987599)

I have been getting this problem for a while now. Im suspecting it's because of throttling, is there any way to get passed this? Using VPN or anything similar?

**Comment by wkvenema** [comment]
Im suffering the same issues. It massively seems to destroy my internet connection.

Hopeyfully there is some solution for this.

**Comment by CptAirwolf** [comment]
Same issue. Need a fix or atleast a reply would be nice!
